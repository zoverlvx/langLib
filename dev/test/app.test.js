import React from "react";
import App from "../js/components/app";
import toJson from "enzyme-to-json";
import {shallow, mount} from "enzyme";

const component = shallow(<App/ >);
const tree = toJson(component);

describe("<App />", () => {
	it("renders 1 <App /> component", () => {
		console.log(tree); 
		expect(component).toHaveLength(1);
	});

	it("provides a snapshot of App's tree", () => { 
		expect(tree).toMatchSnapshot();
	});

	it("", () => {
        
	});
});

//What ought to be tested