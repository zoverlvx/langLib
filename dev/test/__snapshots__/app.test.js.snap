// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> provides a snapshot of App's tree 1`] = `
<div>
  <Route
    component={[Function]}
    path="/"
  />
  <Route
    component={[Function]}
    path="(:language)"
  />
  <Route
    component={[Function]}
    path="(:language)/(:part-of-speech)"
  />
</div>
`;
